### ISSUE TITLE
PHP 8 Support

### ISSUE THREAD

**Issue by marin246** [psliwa/image-optimizer#81]
is PHP8 support in the works? Thanks in advance!

**Comment by mad-briller** [comment]
https://github.com/psliwa/image-optimizer/pull/80

i've added this pr which seems to work fine in php8.1 using symfony 6 components
just waiting for the maintainer to merge it, although the repo seems like it's abandoned and might need to be forked

**Comment by Flynsarmy** [comment]
I also submitted 2 PRs before noticing this repo may be abandoned so I've forked and implemented a bunch of improvements here https://github.com/Flynsarmy/image-optimizer

So far:
* PHPStan level 6 compliance
* PSR-12 coding standards
* Latest version of Symfony components
* Compatibility with svgo 2.8.0
* Moved from TravisCI to Github Actions
* Better documentation inside classes
* Added a method to view which optimizers are installed on your system

My version of the package is only compatible with PHP 8.0+. If you're on an outdated PHP, use this repo instead.

**Comment by mad-briller** [comment]
yeah there is the `spatie/image-optimizer` library that i've moved to instead of this one.
